### DISCUSSION TITLE
Strain softening Mohr-Coulomb Constitutive Model

### DISCUSSION THREAD

**Discussion by Shachen75** [Anura3D/Anura3D_OpenSource#31] [General]
Dear all,

In current Anura3D code (2022 version), we only have linear elastic and Mohr-Coulomb standard soil model. It seems that strain softening model was only available in Anura3D 2017 version. 

Can anyone advise where I can download the strain softening Mohr-Coulomb constitutive model? Should I write a Fortran file and then compile it in Visual Studio? If yes, do we have any guideline or reference on how to write a constitutive model for Anura3D?

Your help is highly appreciated.

**Comment by ayerrocolom** [comment]
Hello,
We only provide linear elastic and Mohr-Coulomb. However, you can code your own external soil model and provide your own dll file following the instructions provided in the tutorial manual, in section 4.5.
I hope this helps.

**Reply by Shachen75** [reply]
Thanks for your help.
